src/SwaggerProvider.Runtime/Parser/Schema.fs

```diff
@@ -145,6 +145,8 @@ type ParameterObject =
       /// Determines the format of the array if type array is used.
       CollectionFormat: CollectionFormat
     }
+    with
+        member x.UnambiguousName = sprintf "%s%A" x.Name x.In
```

Comment:
Does this method generate nice names? What do you think about
```fsharp
sprintf "%sIn%A" x.Name x.In
```

Reply:
That's a nicer name : thumbsup: